spec/element-spec.ts

```diff
@@ -174,7 +182,8 @@ describe("TerminalElement", () => {
   it("restartPtyProcess() check ptyProcessRunning set to true", async () => {
     const newPtyProcess = jasmine.createSpyObj("ptyProcess", ["kill", "write", "resize", "on", "removeAllListeners"])
     newPtyProcess.process = jasmine.createSpy("process").and.returnValue("sometestprocess")
-    nodePty.spawn.and.returnValue(newPtyProcess)
+    const spawn = (nodePty.spawn as jasmine.Spy)
+    spawn.and.returnValue(newPtyProcess)
```

Comment:
I am not great at typescript so I don't know the best way to do this but I think I found a bug in typescript (or I just don't know how typescript works).

If I write this line as `(nodePty.spawn as jasmine.Spy).and.returnValue(newPtyProcess)` it fails but if I save `(nodePty.spawn as jasmine.Spy)` as a variable first it passes.

I'm not sure what is going on there.

The typescript checks should be cleaned up in this whole project.

Reply:
I will take a look at this tonight.

Reply:
I figured it out 🤦‍♂️. This is one of those situations where semicolons would have helped. Javascript thought `(nodePty.spawn as jasmine.Spy)` was a function call for the previous statement.

Reply:
I cleaned up the tests to use non-null assertions instead of type casting